lib/src/golden_test_adapter.dart

```diff
@@ -301,3 +309,98 @@ class FlutterGoldenTestAdapter extends GoldenTestAdapter {
+/// cause localizations to not be loaded properly. This widget sets up the bare
+/// minimum to get the test to run.
+///
+/// Exposed for internal testing. Do not use this explicitly.
+/// {@endtemplate}
+@protected
+@visibleForTesting
+class FlutterGoldenTestWrapper extends StatelessWidget {
+  /// {@macro _flutter_golden_test_wrapper}
+  const FlutterGoldenTestWrapper({
+    super.key,
+    this.globalConfigTheme,
+    this.variantConfigTheme,
+    this.obscureFont = false,
+    required this.child,
+  });
+
+  /// The theme provided by the global [AlchemistConfig], if any.
+  ///
+  /// See [MaterialApp.theme] for more details.
+  final ThemeData? globalConfigTheme;
+
+  /// The theme provided by the current variant's [GoldensConfig], if any.
+  ///
+  /// See [MaterialApp.theme] for more details.
+  final ThemeData? variantConfigTheme;
+
+  /// Whether the default font family of the resolved theme should be set to an
+  /// obscured font.
+  ///
+  /// See [GoldenTestThemeDataExtensions.applyObscuredFontFamily] for more
+  /// details.
+  final bool obscureFont;
+
+  /// The root widget to wrap.
+  ///
+  /// See [MaterialApp.home] for more details.
+  final Widget child;
+
+  /// Resolves the appropriate theme to use for the current test.
+  ///
+  /// If [obscureFont] is true, the default font family of the resolved theme
+  /// will be set to an obscured font. (See
+  /// [GoldenTestThemeDataExtensions.applyObscuredFontFamily] for more details.)
+  ///
+  /// The returned theme will have its text packages stripped. (See
+  /// [GoldenTestThemeDataExtensions.stripTextPackages] for more details.)
+  ///
+  /// The algorithm is as follows:
+  /// - If a [variantConfigTheme] is provided (through a [GoldensConfig]), use
+  ///   it.
+  /// - Otherwise, if a theme is provided through an [InheritedTheme] (such as
+  ///   through an ancestor [MaterialApp] or [Theme] widget), use it.
+  /// - Otherwise, if a [globalConfigTheme] is provided through an
+  ///   [AlchemistConfig], use it.
+  /// - Otherwise, use the [ThemeData.fallback].
+  ThemeData _resolveThemeOf(BuildContext context) {
+    final hasInheritedTheme =
+        context.findAncestorWidgetOfExactType<Theme>() != null;
+    final inheritedTheme = !hasInheritedTheme ? null : Theme.of(context);
+    final fallbackTheme = ThemeData.fallback();
+
+    var resolvedTheme = variantConfigTheme ??
+        inheritedTheme ??
+        globalConfigTheme ??
+        fallbackTheme;
+
+    if (obscureFont) {
+      resolvedTheme = resolvedTheme.applyObscuredFontFamily();
+    }
+
+    return resolvedTheme.stripTextPackages();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return MediaQuery(
+      data: MediaQuery.maybeOf(context) ??
+          MediaQueryData.fromWindow(WidgetsBinding.instance.window),
+      child: Directionality(
+        textDirection: Directionality.maybeOf(context) ?? TextDirection.ltr,
```

Comment:
Because it is the default in the western world. Any thoughts on a different default? 👀

Reply:
I was just wondering. I know is the default in the western world, I'm curious to know if the MaterialApp also defaults it to ltr or if it does any other calculations to determine what to default it to. 🤔

Reply:
It looks like `MaterialApp` uses `WidgetsApp` which uses `Localizations` which in turn creates a default `Directionality`. Copying over this logic or otherwise re-implementing it seems a tad difficult and wasteful in my opinion, so though we could do it, I'd advise to just use `.ltr` for the time being, since that's what most people will use anyway.

Additionally, people can override it if need be, or we could include it in the `AlchemistConfig` as a last resort.
That sound good to you? 👀

Reply:
I'm fine to have it as it is for now. 

It can be done in a different PR. I think, ideally, we should aim to do it the same way Flutter does it.